include/glow/Graph/Graph.h

```diff
@@ -513,6 +513,21 @@ class Function final : public Named {
   LengthsToRangesNode *createLengthsToRanges(llvm::StringRef name,
                                              NodeValue lengths);
 
+  /// Implements an operation that converts the sparse representation given by
+  /// the pair of \p indices and \p values into a dense representation.
+  /// This representation contains each vector of \p values at the corresponding
+  /// index given by \p indices. All indices that are not present in \p indices
+  /// are filled with zeroes. \p indices can contain duplicates, and in this
+  /// case, the corresponding vectors in \p values are added.
+  ///
+  /// The first dimension of the output is the maximum index seen in
```

Comment:
Please change this comment the same way you changed doc for SparseToDenseNode. Let's not say "vector" here. And not mention maximum index, since we don't do it.